.env_sample

```diff
@@ -1 +1 @@
-SENDGRID_API_KEY=''
+SENDGRID_APIKEY=''
```

Comment:
Revert this env var name change. This is the standard `SENDGRID_API_KEY` env var used in multiple sendgrid helper languages.

Reply:
Hi @eshanholtz,

please take a look Samples projects

If we use `SENDGRID_API_KEY` as environment configuration key, in [.NET Platform configuration API](https://docs.microsoft.com/aspnet/core/fundamentals/configuration/), we need use `Configuration["SendGrid:Api:Key"]` to retrieve the APIKey, and JSON configuration will became following example

```json
{
  "SendGrid": {
    "Api":{
     "Key": "CHANGE_ME"
    }
  }
}
```

It create unnecessary depth for configurations.

Reply:
@akunzai Isn't it double underscores that create nesting? i.e. `SendGrid__Api__Key` would create a nested structure like you described, whereas `SendGrid_Api_Key` would simply read from a top-level JSON object? 

I personally think having it nested at one level of depth is more practical for most real-world scenarios, as the `SendGrid` configuration section could then hold all configuration related to SG.

Reply:
@aevitas is right, double underscores create nesting Environment configuration! my bad.

I will drop the commit that Renaming `SENDGRID_API_KEY`